### ISSUE TITLE
Make search field bigger

### ISSUE THREAD

**Issue by kugiigi** [ubports/unity8#371]
### Description
I think the search field is too small for something that's often used. On Unity 7, it's a bit bigger too and I like it.
It would also help for hitting it easier with touch.

**Comment by dobey** [comment]
Did you change the scaling on your device? The field is exactly the same size as when hitting search in Contacts app:

![searchbars](https://user-images.githubusercontent.com/6557070/114197096-03ff5380-9920-11eb-9a23-98c5609df8e2.png)

**Comment by kugiigi** [comment]
Yes it is since it's the default height of text fields in UITK. But I think the search field is a vital part of the app drawer and it should be a bit bigger for visual and interaction purposes. This is of course jut my opinion. I use the search field almost always when using the app drawer.

**Comment by cibersheep** [comment]
It's true that the only field higher than UITK is the one in lock screen. Instead of make search in Dash higher, maybe leave more space as your complaint is that is not easy to hit.

![imatge](https://user-images.githubusercontent.com/6640041/114561489-e575c100-9c6d-11eb-9110-2fa054724c07.png)

**Comment by kugiigi** [comment]
Well, harder to hit is one of my concern but I also think it should stand out a bit visually. Putting more margins doesn't look good in my opinion.
